### PR TITLE
seadatacloud: Update files with those used in production

### PR DESCRIPTION
**Pull request by gregcorbett**
i.e. those deployed on the cineca infrastructure at the end of the project.

### CODE REVIEW

File: dashboards/seadatacloud/container_accounting.json [hunk 1]
```diff
@@ -0,0 +1,1145 @@
...
+      "title": "SeaDataCloud",
+      "tooltip": "",
+      "type": "link",
+      "url": "https://www.seadatanet.org/About-us/SeaDataCloud"
+    }
+  ],
+  "panels": [
+    {
+      "collapsed": false,
+      "datasource": null,
```
Comment (tofu-rocketry) [inline dashboards/seadatacloud/container_accounting.json:39]: Null for datasource. Is this just being used as a title bar?

File: provisioning/datasources/seadatacloud.yml.template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+datasources:
+  - name: Elasticsearch
+    type: elasticsearch
+    url: https://${hostname}:9200
```
Comment (tofu-rocketry) [inline provisioning/datasources/seadatacloud.yml.template:6]: Where's `hostname` come from?

File: provisioning/datasources/seadatacloud.yml.template [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    database: "[accounting-records-*-]YYYY.MM.DD"
```
Comment (tofu-rocketry) [inline provisioning/datasources/seadatacloud.yml.template:7]: This an Elastic thing?

**Review by tofu-rocketry — COMMENTED** [review]
Not sure there's much meaningful review I can do other than checking it doesn't conflict with the other dashboards. Left a few questions.

File: dashboards/seadatacloud/container_accounting.json (hunk 1 above)
Comment (gregcorbett) [reply dashboards/seadatacloud/container_accounting.json:39]: Think so? Think it refers to this row/title bar?

image

File: provisioning/datasources/seadatacloud.yml.template (hunk 2 above)
Comment (gregcorbett) [reply provisioning/datasources/seadatacloud.yml.template:6]: it has to be manually changed to an actual hostname - it's documented here: https://wiki.eduuni.fi/display/EUDATCDI/Accounting+services

File: provisioning/datasources/seadatacloud.yml.template (hunk 3 above)
Comment (gregcorbett) [reply provisioning/datasources/seadatacloud.yml.template:7]: Yeah, it's a wildcard match for indices.

File: dashboards/seadatacloud/container_accounting.json (hunk 1 above)
Comment (tofu-rocketry) [reply dashboards/seadatacloud/container_accounting.json:39]: Oh, collapsible sections. That's neat.

File: provisioning/datasources/seadatacloud.yml.template (hunk 2 above)
Comment (tofu-rocketry) [reply provisioning/datasources/seadatacloud.yml.template:6]: Ok. Maybe stick a comment in about that?

File: provisioning/datasources/seadatacloud.yml.template (hunk 3 above)
Comment (tofu-rocketry) [reply provisioning/datasources/seadatacloud.yml.template:7]: Ok.

File: provisioning/datasources/seadatacloud.yml.template (hunk 2 above)
Comment (gregcorbett) [reply provisioning/datasources/seadatacloud.yml.template:6]:
```suggestion
    # "${hostname}" must be replaced with a relevant value
    url: https://${hostname}:9200
```

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**